charts/opensearch/values.yaml

```diff
@@ -367,3 +367,9 @@ fsGroup: ""
 ## Also see: https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/
 sysctl:
   enabled: false
+
+## Enable to add 3rd Party / Custom plugins not offered in the default OpenSearch image.
+plugins:
+  enabled: false
+  installList: []
+  # - https://github.com/aparo/opensearch-prometheus-exporter/releases/download/1.0.0/prometheus-exporter-1.0.0.zip
```

Comment:
IMO we should not give this as an example and rather keep it empty and add instructions for installation in the README. Many of them can think it as an official recommendation as well.  Rest of the changes looks good to me.

Reply:
@TheAlgo I thought this might be the case. The problem is it'd be nice to have an official example, but I can't find any. haha